ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml.cs

```diff
@@ -39,21 +39,29 @@ public ClinicalDetailsModel(INotificationService service, NtbsContext context, I
             this.auditService = auditService;
         }
 
-        public override async Task<IActionResult> OnGetAsync(int? id)
+        public override async Task<IActionResult> OnGetAsync(int? id, bool isBeingSubmitted)
```

Comment:
Do we want to default `isBeingSubmitted` to false? Or does lack of a parameter do that for us?

Reply:
I am calling this abstract method in `NotificationBase` as well. Since the calls are made when user Validate and Saves. I had two options: just pass false, or have it as optional parameter. Optional one seemed neater.